var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -3,16 +3,16 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
-# Although zlib comes with a configure script, it does not use Autotools
-# The AutotoolsPackage causes zlib to fail to build with PGI
 import glob
 import os
 
+from spack.build_systems.windows_variants import WindowsPackage
 from spack.package import *
 
 
-class Zlib(Package):
+# Although zlib comes with a configure script, it does not use Autotools
+# The AutotoolsPackage causes zlib to fail to build with PGI
+class Zlib(Package, WindowsPackage):
```

Comment:
The only thing `WindowsPackage` provides is `staticmt`. Since this package doesn't use that variant here, does it in fact need to inherit `WindowsPackage`?